systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,394 @@
+        for (KafkaTopic kt : ktl) {
+            cmdKubeClient().deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
+        }
+    }
+
+    @ParallelTest
+    void testThrottlingQuotasDeleteTopic(ExtensionContext extensionContext) {
+        final String kafkaUsername = kafkaUsernamePrefix + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String createAdminName = "create-admin-client" + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String deleteAdminName = "delete-admin-client" + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String topicNamePrefix = classTopicPrefix + "-delete";
+        int topicsCountOverQuota = 500;
+        setupKafkaUserInNamespace(extensionContext, kafkaUsername);
+
+        // Create many topics
+        KafkaAdminClient createAdminClientJob;
+        String createPodName;
+        int offset = 100;
+        for (int i = 0, j = 0; i < topicsCountOverQuota; i = i + offset, j++) {
+            LOGGER.info("Executing {}/{} iteration.", j + 1, topicsCountOverQuota / offset);
+            createAdminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(100)
+                .withTopicOffset(i)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.CREATE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+            resourceManager.createResource(extensionContext, true, createAdminClientJob.defaultAdmin().build());
+            ClientUtils.waitForClientSuccess(createAdminName, NAMESPACE, 10);
+            createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+            PodUtils.waitUntilMessageIsInPodLogs(createPodName, "All topics created", Duration.ofMinutes(6).toMillis());
+            JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+        }
+
+        // Test delete all topics at once - should fail on Throttling Quota limit
+        KafkaAdminClient deleteAdminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(deleteAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicsCountOverQuota)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.DELETE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+        resourceManager.createResource(extensionContext, deleteAdminClientJob.defaultAdmin().build());
+        PodUtils.waitUntilPodIsPresent(NAMESPACE, deleteAdminName);
```

Comment:
Why do you use this instead of a proper solution like in https://github.com/strimzi/strimzi-kafka-operator/pull/5509 ?

Reply:
Maybe you should even wait until the job is completed in that case?

Reply:
Removed all those lines (which currently fix tests). 
Waiting for #5509 to be merged and my tests will work without this.